src/plugins/presentation_panel/public/panel_component/presentation_panel_internal.tsx

```diff
@@ -68,8 +68,8 @@ export const PresentationPanelInternal = <
   );
   const viewMode = rawViewMode ?? 'view';
 
-  const [initialLoadComplete, setInitialLoadComplete] = useState(!dataLoading);
-  if (!initialLoadComplete && (dataLoading === false || (api && !api.dataLoading))) {
+  const [initialLoadComplete, setInitialLoadComplete] = useState(false);
```

Comment:
This change makes the initial loading spinner appear until after the first data has been loaded. This only applies if the panel's API contains a `dataLoading` subject.